### PR TITLE
Vaishnavi_Assignment_34_Done

### PR DESCRIPTION
**Pull request by VaishnaviVJun20**
Program1:

image

Program 2:

image

### CODE REVIEW

File: src/vaishnaviVJun20_Collection/ReplaceString.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+public class ReplaceString {
+
+	void replaceGivenStringInArrayList(ArrayList<String> arrList, String strToReplace, String strReplaceWith) {
+		int count = 0;
+		System.out.print("Techno word is present in a given arrayList at index: ");
+		while (arrList.contains(strToReplace)) {
+			int strIndex = arrList.indexOf(strToReplace);
+			System.out.print(strIndex + ",");
+			arrList.set(strIndex, strReplaceWith);
```
Comment (harshvegada) [inline src/vaishnaviVJun20_Collection/ReplaceString.java:22]: You can add counter in while loop to check Total number of word to replaced

**Review by harshvegada — APPROVED** [review]
Both Questions Logic is expected,
No changes required
keep it up

**Review by harshvegada — APPROVED** [review]

File: src/vaishnaviVJun20_Collection/ReplaceString.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+		}
+		System.out.println("\nOutput arrayList is: " + arrList);
+
+		for (int index = 0; index < arrList.size(); index++) {
```
Comment (harshvegada) [inline src/vaishnaviVJun20_Collection/ReplaceString.java:26]: Here you can handle while loop logic in this for as well, like below, jsut FYI
image

**Review by KrishnaKTechnocredits — APPROVED** [review]
Code Looks Good to me.

**Pull request merged**